### DISCUSSION TITLE
请问什么功能需要请求macOS客户端辅助功能权限

### DISCUSSION THREAD

**Discussion by hywhuangyuwei** [qier222/YesPlayMusic#1224] [Q&A]
![image](https://user-images.githubusercontent.com/33554665/148915511-9e88ec7a-442f-40be-87f5-3c433e73754c.png)

**Comment by pan93412** [comment]
同為 macOS 使用者，我是選擇拒絕。彈出原因未知，但我不記得 YPM 哪裡有用到輔助功能。cc @qier222

**Comment by qier222** [comment]
可能是在选择音频输出设备的时候用得到？或者是全局快捷键？
